_data/ganeshShetty98/ganeshShetty98.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Shetty Ganeshprasad
+institution: SDM College of Engineering and Technology 
+profile_pic: ganesh.png 
+quote: Thanks to all teachers and my dear friends who encouraged and motivated me to get here. I'm going to miss you all!!!
```

Comment:
Character count exceeds the limit; 100. You might want to modify this

Reply:
Yes  i have changed as u suggested and have replaced my image with new current one

Reply:
okay thanks, I corrected it thanks.

Reply:
if everything correct and reviewed why can't I merge.

Reply:
You are fine don't worry about it, the merging team will handle that part

Reply:
Okay Thanks for info.